docs/release-notes.md

```diff
@@ -33,6 +33,9 @@
 * [ZNTA-928](https://zanata.atlassian.net/browse/ZNTA-928) - Readonly project doesn't have "lock" icon in UI
 * [ZNTA-793](https://zanata.atlassian.net/browse/ZNTA-793) - Remove unused method in TranslationMemoryAction
 
+<h5>Infrastructure Changes</h5>
+* Zanata now requires JBoss EAP 6.4.6.GA or later (recommended), or WildFly version 10.x.
```

Comment:
Well, the idea is that EAP is recommended (6.4.6 or later), but WildFly 10
is also... possible.

On 16 March 2016 at 13:51, Damian Jansen <notifications@github.com> wrote:

> In docs/release-notes.md
> <https://github.com/zanata/zanata-server/pull/990#discussion_r56278787>:
>
> > @@ -33,6 +33,9 @@
> >  * [ZNTA-928](https://zanata.atlassian.net/browse/ZNTA-928) - Readonly project doesn't have "lock" icon in UI
> >  * [ZNTA-793](https://zanata.atlassian.net/browse/ZNTA-793) - Remove unused method in TranslationMemoryAction
> >
> > +<h5>Infrastructure Changes</h5>
> > +* Zanata now requires JBoss EAP 6.4.6.GA or later (recommended), or WildFly version 10.x.
>
> Requires, but recommended? Or is "later than 6.4.6.GA" recommended?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/zanata/zanata-server/pull/990/files/5ebc7bcb5d17ad1f2421243ecd6379f78e6142fd#r56278787>
>



-- 
Sean Flanigan

Principal Software Engineer
Globalisation Tools Engineering
Red Hat